pkg/util/helper/binding.go

```diff
@@ -89,18 +89,18 @@ func GetBindingClusterNames(targetClusters []workv1alpha2.TargetCluster, binding
 func FindOrphanWorks(c client.Client, bindingNamespace, bindingName string, clusterNames []string, scope apiextensionsv1.ResourceScope) ([]workv1alpha1.Work, error) {
 	var needJudgeWorks []workv1alpha1.Work
 	if scope == apiextensionsv1.NamespaceScoped {
-		workList, err := GetWorksByLabelsSet(c, labels.Set{
```

Comment:
Should we remove the old `GetWorksByLabelsSet` as well? Otherwise, it might work as unexpected.

Reply:
https://github.com/karmada-io/karmada/blob/cee1a2f4a16b5292b61007ce3f87e887c1e8acc2/pkg/controllers/federatedresourcequota/federated_resource_quota_status_controller.go#L130-L133
There is a different usage here.